src/index.js

```diff
@@ -100,7 +115,14 @@ function configureToMatchImageSnapshot({
       ({ pass } = result);
 
       if (!pass) {
-        updateSnapshotState(snapshotState, { unmatched: snapshotState.unmatched + 1 });
+        const currentRun = timesCalled.get(snapshotIdentifier);
+        if (retryTimes && (currentRun <= retryTimes)) {
+          // eslint-disable-next-line no-console
+          console.warn(`${currentTestName} failed, retrying ${(retryTimes + 1) - currentRun} more time(s)`);
```

Comment:
Do other Jest matchers do this too? Not sure if we want to log this ourselves. Does Jest not provide some type of visual guidance when a test is being re-ran?

Reply:
I removed the console.warn as it does seems weird to do it in an plugin. They don't currently provide any guidance, but I'm not sure that it's necessary. I originally put it there more to see that it's working for myself....